content/slice/encoding/constructed-types.md

```diff
@@ -14,6 +26,24 @@ compact struct Pair { key: Key, value: Value }
 
 `Key` represents the dictionary's key type and `Value` represents the dictionary's value type.
 
+{% slice2 %}
+
+## Result
+
+A `Result<Success, Failure>` is encoded exactly like a compact enum with two enumerators (`Success` and `Failure`),
+where each enumerator has a single field of the Success resp. Failure type.
```

Comment:
Honestly, I find it perfectly readable, and better than the suggestion... But you don't see this `resp.` syntax used much nowadays. I think just writing it out would be fine.
```suggestion
where each enumerator has a single field of the respective (Success or Failure) type.
```